LONG_DESCRIPTION.rst

```diff
@@ -1,7 +1,7 @@
 Official Ably Bindings for Python
 ==================================
 
-A Python client library for ably.io realtime messaging
+A Python client library for ably realtime messaging
```

Comment:
We should always start the Ably brand with a capital letter. Also, this looks like a sentence so perhaps it should end in a full stop, while you're in the area. 😉 

```suggestion
A Python client library for Ably Realtime messaging.
```

We will write this up soon under https://github.com/ably/engineering/issues/19.